src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -33,8 +33,8 @@
 _STREAMING_OUTPUT_CALL_METHOD = '/grpc.testing.TestService/StreamingOutputCall'
 
 _INVOCATION_METADATA = (
-    ('initial-md-key', 'initial-md-value'),
-    ('trailing-md-key-bin', b'\x00\x02'),
+    ('x-grpc-test-echo-initial', 'initial-md-value'),
+    ('x-grpc-test-echo-trailing-bin', b'\x00\x02'),
```

Comment:
an example of the friction that I was commenting before, we had to modify unit tests because the server was being used by two different environments.

Reply:
I overlooked this metadata key in the first place. There shouldn't be more upcoming conflicts between interop test and unit tests.